packages/components/src/components/input/input.ts

```diff
@@ -471,68 +549,190 @@ export default class SdInput extends SolidElement implements SolidFormControl {
               @keydown=${this.handleKeyDown}
               @focus=${this.handleFocus}
               @blur=${this.handleBlur}
-            />
-
-            ${hasClearIcon
-        ? html`
+            >
+            <!-- TODO: substitute text-neutral-400 for text-neutral-500 when available -->
```

Comment:
Did you open up a ticket for that? That would be great.

Reply:
https://github.com/solid-design-system/figma/issues/155

Just read that there already is an issue. :)